Add Discogs singleton limitation to docs (#4631)
beetbox/beets: docs/plugins/discogs.rst

==============

The ``discogs`` plugin extends the autotagger's search capabilities to

> include matches from the `Discogs`_ database when importing albums.
> Note that matching of singleton tracks using the ``discogs`` plugin
> is not currently supported.

Reviewer: ```suggestion
include matches from the `Discogs`_ database when importing albums.
(The plugin does not yet support matching singleton tracks.)
```

Just a little simpler.

Author: I agree, changed in latest commit.

* Make sure that your system clock is accurate. The Discogs servers can reject
  your request if your clock is too out of sync.

> Furthermore, the plugin itself is limited to matching albums, so no

Reviewer: ```suggestion
The plugin can only match albums, so no
```

Slightly more direct wording.

Author: I agree, changed in latest commit.
